examples/lock-app/nrfconnect/README.md

```diff
@@ -1,53 +1,101 @@
+the remote access, application also allows controlling lock and device states
+using buttons and visualizes those states on the LEDs.
 
 <hr>
 
--   [CHIP nRF52840 Lock Example Application](#chip-nrf52840-lock-example-application)
-    -   [Introduction](#introduction)
-    -   [Device UI](#device-ui)
-    -   [Building](#building)
-        -   [Using Docker container](#using-docker-container)
-        -   [Using Native shell](#using-native-shell)
-        -   [Supported nRF Connect SDK versions](#supported-nrf-connect-sdk-versions)
-    -   [Configuring the example](#configuring-the-example)
-    -   [Flashing and debugging](#flashing-and-debugging)
-    -   [Accessing the command line](#accessing-the-command-line)
+-   [Overview](#overview)
+-   [Requirements](#requirements)
+-   [Device UI](#device-ui)
+-   [Building](#building)
+    -   [Using Docker container](#using-docker-container)
+    -   [Using Native shell](#using-native-shell)
+    -   [Supported nRF Connect SDK versions](#supported-nrf-connect-sdk-versions)
+-   [Configuring the example](#configuring-the-example)
+-   [Flashing and debugging](#flashing-and-debugging)
+-   [Accessing the command line](#accessing-the-command-line)
 
 <hr>
 
-<a name="intro"></a>
+<a name="overview"></a>
 
-## Introduction
+## Overview
 
-![nrf52840 DK](../../platform/nrfconnect/doc/images/nrf52840-dk.jpg)
+This example application is running on the nRF Connect platform, which is based
+on the
+[nRF Connect SDK](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/index.html)
+and [Zephyr RTOS](https://zephyrproject.org/). Visit CHIP's
+[nRF Connect Platform Overview](TODO:...) to read more information about
+platform structure and dependencies.
 
-The nRF52840 lock example application provides a working demonstration of a
-connected door lock device, built using CHIP, and the Nordic nRF Connect. The
-example supports remote access and control of a simulated door lock over a
-low-power, 802.15.4 Thread network. It is capable of being paired into an
-existing CHIP network along with other CHIP-enabled devices. The example targets
-the
-[Nordic nRF52840 DK](https://www.nordicsemi.com/Software-and-Tools/Development-Kits/nRF52840-DK)
-development kit, but is readily adaptable to other nRF52840-based hardware.
+CHIP device running lock application is controlled by the CHIP controller device
+over Thread protocol. It is assumed that CHIP device by default has Thread
+disabled, so it should be paired with CHIP controller and get configuration from
+it. There are few actions necessary to make before full communication is
+possible and they were described below. There is also a test mode, which allows
+to start Thread with default settings, by pressing button manually, but it does
+not fully guarantee that device will be able to communicate with CHIP controller
+and other devices.
+
+### BLE Advertising
+
+After powering up device for the first time, it should start advertising over
```

Comment:
I think this text should describe specifically this example, so there's no need to mention that other devices may start advertising automatically.